### PR TITLE
feat(migrations): Defining db schema and adding movies data

### PR DESCRIPTION
**Pull request by shraddha1704**
### What
Defining database schema and adding data for migration

### Why
Miyagi training

### CODE REVIEW

File: db/migrations/20190313145917_create_movies.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    return Knex.schema.createTable('movies', (table) => {
+    table.increments('id').primary();
+    table.text('title').notNullable();
+    table.integer('release_year');
+    });
+};
+
+exports.down = (Knex) =>  {
+    return Knex.schema.dropTable('movies');
+};
```
Comment (jesscxu) [inline db/migrations/20190313145917_create_movies.js]: nit: i think you need to add a new line?

File: db/seeds/index.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const Movies = require('./data/movies');
+
+exports.seed = async (Knex) => {
+  await Knex('movies').truncate();
+  await Knex('movies').insert(Movies);
+};
```
Comment (jesscxu) [inline db/seeds/index.js]: nit: same here

File: db/migrations/20190313145917_create_movies.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+exports.up = (Knex) => {
```
Comment (jesscxu) [inline db/migrations/20190313145917_create_movies.js]: can you convert this to `async/await`?

**Review by ajpawlicki — APPROVED** [review]

**Review by jesscxu — APPROVED** [review]
LGTM

**Pull request merged**